src/opera_disp_tms/generate_sw_disp_tile.py

```diff
@@ -69,39 +72,70 @@ def find_needed_granules(frame_ids: Iterable[int], begin_date: datetime, end_dat
         frame_ids: The frame ids to generate the tile for
         begin_date: Start of secondary date search range to generate tile for
         end_date: End of secondary date search range to generate tile for
+        strategy: Selection strategy for granules within search range ("minmax" or "all")
```

Comment:
Is the strategy for the search range of the dates? 
```suggestion
        strategy: Selection strategy for granules within search date range ("max", "minmax" or "all")
```